backend/tests/unit/kyc.test.js

```
@@ -0,0 +1,435 @@
+        monthlySendLimit: 5000.00,
+        currentMonthSent: 5000.00,
+        resetDate: new Date('2024-01-01')
+      });
+
+      // Simulate new month
+      const newMonthDate = new Date('2024-02-01');
+      const originalDate = Date;
+      global.Date = class extends Date {
+        constructor() {
+          return new originalDate(newMonthDate);
+        }
```

Biome (2.1.2)

[error] This constructor has code paths that return without calling `super()` first.

This statement returns from the constructor before `super()` has been called:

If this is intentional, add an explicit throw statement in unsupported paths.

(lint/correctness/noUnreachableSuper)